main.go

```diff
@@ -91,6 +105,18 @@ func main() {
 	// Parse command-line arguments.
 	args := argsParse(nil)
 
+	// Read the config file
+	err := configParse(args.configFile)
+	if err != nil {
+		stderrLogger.Printf("Failed to read config file: %v", err)
```

Comment:
Error of parsing function is a "read error" instead of a "parsing error"? I know it's a nit-picking, but there's also small code duplication issue there in the parsing - see other comments

Reply:
I get the point, I'll move the error message generation to configParse()